src/main/java/pl/arturzgodka/controllers/SkillsController.java

```diff
@@ -0,0 +1,94 @@
+package pl.arturzgodka.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import pl.arturzgodka.datamodel.HeroSkillDataModel;
+import pl.arturzgodka.jsonmappers.SkillMapper;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Controller
+public class SkillsController {
+
+    private final List<String> heroClasses = Arrays.asList("Barbarian", "Crusader", "Demon Hunter", "Monk", "Necromancer", "Witch Doctor", "Wizard");
```

Comment:
Either put those in dedicated classes/database/files. These are static resources, no point to do it this way

Reply:
Now these lists are in separate class as public static fields.

Reply:
done